client/source/js/modules/help/index.js

```diff
@@ -6,7 +6,10 @@ define(['angular', 'ui.router'], function (angular) {
       $stateProvider
         .state('help', {
           url: '/help',
-          templateUrl: 'js/modules/help/help.html'
+          templateUrl: 'js/modules/help/help.html',
+          controller: ['$scope', function ($scope) {
```

Comment:
@nikgraf You don't need to use array inline annotation as there is ngAnnotate task run before code is minified in gulp ( https://github.com/olov/ng-annotate )